extern/agg24-svn/include/agg_alpha_mask_u8.h

```diff
@@ -1,6 +1,6 @@
 //----------------------------------------------------------------------------
 // Anti-Grain Geometry - Version 2.4
-// Copyright (C) 2002-2005 Maxim Shemanarev (http://www.antigrain.com)
+// Copyright (C) 2002-2005 Maxim Shemanarev (http://antigrain.com/)
```

Comment:
Everything in `extern` is an external source; you should revert all of these, and send them upstream first.